src/libs/runtime-utils.js

```diff
@@ -196,17 +196,25 @@ export const isCurrentGalaxyDiskDetaching = apps => {
   return currentGalaxyApp && _.includes(currentGalaxyApp.status, ['DELETING', 'PREDELETING'])
 }
 
-export const collapsedRuntimeStatus = runtime => {
```

Comment:
I disliked this function name having `collapsed` as a non-description. Renamed to be consistent with the one we have for apps that essentially does the same thing.